test_cases/README.md

```diff
@@ -1,31 +1,35 @@
 ## Regression tests for typeshed
 
-This directory contains regression tests for the stubs found elsewhere in the
-typeshed repo.
+This directory contains code samples that act as a regression test for the
+standard library stubs found elsewhere in the typeshed repo.
 
-**This directory should *only* contain tests for functions and classes which
+**This directory should *only* contain test cases for functions and classes which
 are known to have caused problems in the past, where the stubs are difficult to
 get right.** 100% test coverage for typeshed is neither necessary nor
 desirable, as it would lead to code duplication. Moreover, typeshed has
 multiple other mechanisms for spotting errors in the stubs.
 
 ### The purpose of these tests
 
-Different tests in this directory serve different purposes. For some stubs in
+Different test cases in this directory serve different purposes. For some stubs in
 typeshed, the type annotations are complex enough that it's useful to have
 basic sanity checks that test whether a type checker understands the intent of
```

Comment:
> basic sanity checks

I'd remove `basic` from this sentence. While "Sanity Check" is not in Merriam Webster's dictionary, Wikipedia says on [Sanity Check](https://en.wikipedia.org/wiki/Sanity_check)'s page that a sanity check is "...a basic test to quickly evaluate", which would make `basic` a redundant adjective.